### DISCUSSION TITLE
How to get issues by specific severity level. eg- Critical, High

### DISCUSSION THREAD

**Discussion by Aliraza-Gulamali** [aquasecurity/tfsec#1660] [Help and suppports]
I have so many issues coming up after successful TfSec scan, Now, want to filter the results based on the severity and configure further TfSec Scan only for Critical and High Severity issues and ignoring rest of the severity issues.

Is it possible to get issue based on specific severity eg- Critical, High  ? 
If yes, how I can achieve this. is there any parameters / arguments available to do this.

**Comment by mijdavis2** [comment]
![image](https://user-images.githubusercontent.com/10160872/163423295-5b61644d-50fb-432e-9ae8-c6045c91c7c6.png)

You can sort, but AFAIK you can't filter the output based on severity. You can be explicit about rule includes and excludes, tho, via `--include` and `--exclude` options. The `--help` arg has solid docs on cli options.

**Comment by liamg** [comment]
You can use the `--minimum-severity` parameter to only show results above a certain severity e.g.
```
tfsec --minimum-severity CRITICAL
```

**Reply by zoro16** [reply]
You can run the following command:

 ```tfsec . | grep -A 20 CRITICAL```

Where `-A` is refering to the number of lines after the line that includes `CRITICAL`. And you can change `CRITICAL` to the severity you are looking for e.g. `HIGH` .

**Reply by zoro16** [reply]
![Screenshot 2022-08-08 at 18 43 32](https://user-images.githubusercontent.com/12529183/183469915-5834e231-500e-4887-b307-68d336ef79ef.png)
